src/Microsoft.Dnx.Tooling/ClearCache/ClearCacheCommand.cs

```diff
@@ -0,0 +1,73 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.IO;
+
+namespace Microsoft.Dnx.Tooling
+{
+    internal class ClearCacheCommand
+    {
+        public Reports Reports { get; }
+        public string HttpCacheDirectory { get; }
+        public ClearCacheCommand(Reports reports, string httpCacheDirectory)
+        {
+            Reports = reports;
+            HttpCacheDirectory = httpCacheDirectory;
+        }
+        
+        public int Execute()
+        {
+            if(Directory.Exists(HttpCacheDirectory))
+            {
+                Reports.Information.WriteLine($"Clearing cache directory {HttpCacheDirectory}");
+
+                Exception exception = null;
+
+                for (var i = 0; i < 3; i++)
+                {
+                    try
+                    {
+                        DeleteDirectoryRecursively(HttpCacheDirectory, false);
+                        Reports.Information.WriteLine("Cache cleared.");
+                        return 0;
+                    }
+                    catch (Exception e)
+                    {
+                        exception = e;
```

Comment:
Never a fan of retries because they can add up to a lot in the event of a flaky system. E.g. running 30 `dnu clearcache` commands (once for each repo let's say), and they might all fail for some reason. With a 0.5sec delay that's 30 repos * 0.5sec * 3 retries = 45 seconds to completely fail.